.github/workflows/mypy-stubtest.yml

```diff
@@ -16,13 +16,7 @@ jobs:
       - name: Set up Python 3
         uses: actions/setup-python@v5
         with:
-          python-version: 3.9
-
-      - name: Install mypy
-        run: |
-          pip3 install -r requirements/testing/mypy.txt \
-            -r requirements/testing/all.txt
-          pip3 install .
+          python-version: 3.11
```

Comment:
I think we do want to run on the oldest supported python version to avoid accidentally allowing typing features that were introduced more recently.

Reply:
👍 done